tests/conftest.py

```diff
@@ -20,9 +20,35 @@
 import pytest
 import pytest_asyncio
 import websockets
+from http_server.test_web_server import StaticWebServer
 from test_helpers import execute_command, get_tree, goto_url, read_JSON_message
 
 
+@pytest_asyncio.fixture(scope='session')
+def static_web_server():
+    # TODO: provide a path to a certificate for HTTPS once `ignoreHTTPSErrors`
+    # capability is implemented.
+    server = StaticWebServer()
+    yield server
+    server.shutdown()
+
+
+@pytest_asyncio.fixture(scope='session')
+def url_200(static_web_server):
```

Comment:
I would also minimize the number of fixtures. IMO, only `url` with parameters is sufficient

Reply:
`url` is sufficient, but having wrappers easies usage. Moved `url` to fixture as well.

Reply:
I am not sure, if you, for example, need 5 different status codes in a test you would end up importing 5 different wrappers. And then again, some folks would just use the more general wrapper and some will use more specific, and then the question is do we add a wrapper for each of the status code, what about other headers, etc?

Reply:
We already have a few helper functions (`execute_command`, `get_tree`, `goto_url`) which can be implemented in-place using other methods. It's helpful if you want to test a common-case scenario. And you can fallback to the lower level function when you need some special behaviour.